jekyll/_cci2/machine-runner-3-configuration-reference.adoc

```diff
@@ -161,18 +164,6 @@ runner:
 === runner.cleanup_working_directory
 `$CIRCLECI_RUNNER_CLEANUP_WORK_DIR`
 
-CircleCI recommends using a knowable directory, however it is possible to specify `%s` in the path. This value will be replaced with a different value for each job. This is a substitution that is only knowable at job runtime, under the environment variable `$CIRCLE_WORKING_DIRECTORY`.
-
-Example:
-
-```yaml
-runner:
-  working_directory: "/var/lib/circleci-runner/workdir"
-  cleanup_working_directory: false
-```
-
-`$CIRCLECI_RUNNER_CLEANUP_WORK_DIR`
```

Comment:
Ah, looks like this bit was basically duplicated before?

Reply:
yes, I found the same content on our Runner 1.x docs:
https://circleci.com/docs/runner-config-reference/#runner-cleanup-working-directory

since `%s` works for Runner 1.x, I believe I didnt need to remove any content from https://circleci.com/docs/runner-config-reference/#runner-cleanup-working-directory 👍